ansible_base/lib/utils/encryption.py

```diff
@@ -70,3 +79,30 @@ def decrypt_string(self, value: str) -> str:
 
 
 ansible_encryption = SimpleLazyObject(func=lambda: Fernet256())
+
+
+class SharedSecretNotFound(Exception):
+    pass
+
+
+def generate_hmac_sha256_shared_secret(nonce: Optional[str] = None) -> str:
+    shared_secret = getattr(settings, 'ANSIBLE_BASE_SHARED_SECRET', None)
+    if shared_secret is None or not shared_secret:
+        logger.error("The setting ANSIBLE_BASE_SHARED_SECRET was not set, insecurely using default")
```

Comment:
So I think this is outdated now :sweat_smile: 

Maybe something like:

```suggestion
        logger.error("The setting ANSIBLE_BASE_SHARED_SECRET was not set, some functionality may be disabled.")
```